constellations/gemini-admin/src/emailProgram/emailPrograms.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
+require("./emailPrograms.sass");
+import axios from "axios";
+import React from "react";
+import API, { executeApiCalls } from "../api.js";
+import { getFullName } from "../utils/userUtils.js";
+
+export class EmailProgramsPage extends React.Component {
+    state = {
+        selectProgramId: "",
+        selectedProgramName: "",
+        programs: [],
+        classes: [],
+        classesForProgram: [],
+        userIds: [],
+        usersInClass: [],
+        selectedUsers: [],
+    };
+
+    componentDidMount = () => {
+        const apiCalls = [
+            API.get("api/programs/all"),
+            API.get("api/classes/all"),
+        ];
+        axios
+            .all(apiCalls)
+            .then(
+                axios.spread((...responses) => {
+                    const programs = responses[0].data;
+                    const classes = responses[1].data;
+                    const hasClassId = responses.length > 3;
+                    let classObj = hasClassId ? responses[3].data : {};
+                    let selectedProgramId = hasClassId
+                        ? classObj.programId
+                        : programs[0].programId;
+
+                    this.setState({
+                        selectProgramId: selectedProgramId,
+                        programs: programs,
+                        classes: classes,
+                    });
+                })
+            )
+            .catch((err) => {
+                console.log("Error: api call failed. " + err.message);
+            });
+    };
+
+    handleProgramChange = (event, value) => {
+        var selectedProgram = this.state.programs.find(
+            (program) => program.programId === event.target.value
```

Comment:
event.target.value is used often here, so I would just save it to a variable first.
```
const programId = event.target.value;

... program.programId === programId
```